### PR TITLE
Add lock icon to permission denied UI page

### PR DESCRIPTION
**Pull request by dbczumar**
## What changes are proposed in this pull request?
 
Adds a lock icon to the permission denied UI page for experiments

Before:

Screen Shot 2019-07-18 at 12 56 40 PM

After:

Screen Shot 2019-07-18 at 12 54 21 PM

*Note*: Content of gray text at the bottom is an example; this text is replaced by the error message retrieved from the backend.
 
## How is this patch tested?

Manual UI testing
 
### Is this a user-facing change? 

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

This is a very minor change that is unlikely to affect most users.
 
### What component(s) does this PR affect?
 
- [X] UI
- [ ] CLI 
- [ ] API 
- [ ] REST-API 
- [ ] Examples 
- [ ] Docs
- [ ] Tracking
- [ ] Projects 
- [ ] Artifacts 
- [ ] Models 
- [ ] Scoring 
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:
 
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/server/js/src/components/PermissionDeniedView.js [hunk 1]
```diff
@@ -11,7 +12,13 @@ export default class PermissionDeniedView extends Component {
     const defaultMessage = 'The current user does not have permission to view this page.';
     return (
       <div>
-        <h1 className="center" style={{ paddingTop: '100px' }}>
+        <img
+          className='center'
+          style={{ height: '300px', marginTop: '80px' }}
```
Comment (Zangr) [inline mlflow/server/js/src/components/PermissionDeniedView.js]:
```suggestion
          style={{ height: 300, marginTop: 80 }}
```

File: mlflow/server/js/src/components/PermissionDeniedView.js [hunk 2]
```diff
@@ -11,7 +12,13 @@ export default class PermissionDeniedView extends Component {
...
+          src={permissionDeniedLock}
+          alt='permission denied'
+        />
+        <h1 className="center" style={{ paddingTop: '10px' }}>
```
Comment (Zangr) [inline mlflow/server/js/src/components/PermissionDeniedView.js]:
```suggestion
        <h1 className="center" style={{ paddingTop: 10 }}>
```

**Review by Zangr — APPROVED** [review]
LGTM with some minor suggestions.

**Comment by dbczumar** [comment]
Thanks @Zangr!

**Pull request merged**